catalog/dags/common/sensors/utils.py

```diff
@@ -35,9 +39,40 @@ def get_most_recent_dag_run(dag_id) -> list[datetime] | datetime:
     return []
 
 
-def wait_for_external_dag(external_dag_id: str, task_id: str | None = None):
+@task
+def get_dags_with_concurrency_tag(
+    tag: str, excluded_dag_ids: list[str], session=None, **context
```

Comment:
Is `**context` necessary here? Can we set `dag=None` directly and use the magic that Airflow provides for injecting that directly?

Reply:
You're right! Done 🚀